BDArmory/UI/BDAEditorArmorWindow.cs

```diff
@@ -1123,6 +1175,290 @@ void Visualize()
+                                        if (isChair != null)
+                                        {
+                                            break; 
+                                        }
+                                        ModuleCommand AIParent = null;
+                                        if (kvp.Value[0].parent) AIParent = kvp.Value[0].parent.FindModuleImplementing<ModuleCommand>();
+                                        if (AIParent == null)
+                                        {
+                                            nonCockpitWM = true;
+                                        }
+                                        break;
+                                    }
+                                case "BDModulePilotAI":
+                                case "BDModuleSurfaceAI":
+                                case "BDModuleVTOLAI":
+                                case "BDModuleOrbitalAI":
+                                    {
+                                        AIs++;
+                                        if (AIs > 1) //only 1 WM per vessel. TODO - remember to change this out if Doc ever gets mothership sub-WMs implemented fully
+                                        {
+                                            if (!string.IsNullOrEmpty(blacklistedParts)) blacklistedParts += " | ";
+                                            blacklistedParts += $"{kvp.Value[0].partInfo.title}(AI: {kvp.Value.Count}/1)";
+                                        }
+                                        //editorLogic.fetch.ship.parts[0] doesn't account for re-rooting the craft. fetch.ship also doesn't support .rootpart
+                                        //if (kvp.Value[0].parent != EditorLogic.fetch.ship.Parts[0] || kvp.Value[0] != EditorLogic.fetch.ship.Parts[0])
+                                        var isChair = kvp.Value[0].FindModuleImplementing<KerbalSeat>();
+                                        if (isChair != null)
+                                        {
+                                            break;
+                                        }
+                                        ModuleCommand AIParent = null;
+                                        if (kvp.Value[0].parent) AIParent = kvp.Value[0].parent.FindModuleImplementing<ModuleCommand>();
+                                        if (AIParent == null)
+                                        {
+                                            nonCockpitAI = true;
+                                        }
+                                        break;
+                                    }
+                                case "ModuleCommand":
+                                case "KerbalSeat":
+                                    {
+                                        int crewCount = kvp.Value[0].FindModuleImplementing<ModuleCommand>().minimumCrew;
```

Comment:
This throws an exception since `KerbalSeat`s don't have a `ModuleCommand`.
Is `KerbalSeat` meant to be a separate `case` section in case those are limited?